aws_lambda_powertools/utilities/parameters/__init__.py

```diff
@@ -0,0 +1,247 @@
+# -*- coding: utf-8 -*-
+
+"""
+Parameter retrieval and caching utility
+"""
+
+from typing import Dict, Optional, Union
+
+import boto3
+from boto3.dynamodb.conditions import Key
+
+from .base import BaseProvider, GetParameterError
+
+__all__ = [
+    "BaseProvider",
+    "GetParameterError",
+    "DynamoDBProvider",
+    "SecretsProvider",
+    "SSMProvider",
+    "get_parameter",
+    "get_parameters",
+    "get_secret",
+]
+
+
+class SSMProvider(BaseProvider):
+    """
+    AWS Systems Manager Parameter Store Provider
+    """
+
+    client = None
+
+    def __init__(
+        self, region: Optional[str] = None,
+    ):
+        """
```

Comment:
Could you add an `example section` in the docstring to show the expected usage? i.e. those who might not have access to the docs (IDE shows it)

Logger example as that's also available in the API reference: https://awslabs.github.io/aws-lambda-powertools-python/api/logging/index.html#example